desktopApp/yarn.lock

```diff
@@ -0,0 +1,2641 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
I just realized this, we don't use yarn on the project, I like yarn, but just to be consistent, can you use `npm` instead?